internal/zinc-core/src/main/scala/sbt/internal/inc/Incremental.scala

```diff
@@ -413,12 +414,21 @@ object Incremental {
             1
           )
         else {
+          val analysis =
+            if (hasSubprojectChange)
+              previous.copy(
+                apis = initialChanges.external.allModified.foldLeft[APIs](previous.apis) {
+                  (apis, clazz) =>
+                    apis.markExternalAPI(clazz, lookup.lookupAnalyzedClass(clazz, None).get)
```

Comment:
Passing `None` is okay? using `.get` is okay?

Reply:
The second parameter was something that got added by @dwijnand recently for potential perf benefit build tools can make. I think it's ok to pass in `None` here.

`.get` on the other hand might not be good. Does something like this make sense? 

```suggestion
                    lookup.lookupAnalyzedClass(clazz, None) match {
                      case Some(ac) => apis.markExternalAPI(clazz, ac)
                      case _        => apis
                    }
```